usagov/vote-gov-nvrf-app
# hardcodes field id's for consistency

Updates inputData id's to be hardcoded and matching with the field id's defined in fieldData

Testing:
1. add value in first name field and apartment number field
2. test error handling
3. check that data is saved and correct on confirm page
4. test editing the inputs as well
